src/main/kotlin/stringAddCalculator/Calculator.kt

```diff
@@ -0,0 +1,46 @@
+package stringAddCalculator
+
+object Calculator {
+    const val NEGATIVE_ERROR = "음수는 계산할 수 없습니다."
+    private const val COMMA_DELIMITER = ","
+    private const val COLON_DELIMITER = ":"
+
+    private val DEFAULT_DELIMITER_REGEX = Regex("$COLON_DELIMITER|$COMMA_DELIMITER")
+    private val CUSTOM_DELIMITER_REGEX = Regex("//(.)\n(.*)")
+
+    fun calculate(input: String?): Int {
+        val checkInput: String = checkInput(input)
+        val splitList = splitCustomDelimiter(checkInput) ?: splitString(checkInput)
+        val convertIntList = convertInt(splitList)
+        return add(convertIntList)
+    }
+
+    fun checkInput(input: String?): String = if (input.isNullOrBlank()) "0" else input
+
+    fun splitString(input: String): List<String> {
+        return input.split(DEFAULT_DELIMITER_REGEX)
+    }
+
+    fun splitCustomDelimiter(input: String): List<String>? {
+        val result = CUSTOM_DELIMITER_REGEX.find(input) ?: return null
```

Comment:
```suggestion
    fun splitCustomDelimiter(input: String): List<String> {
        val result = CUSTOM_DELIMITER_REGEX.find(input) ?: return emptyList()
```
컬렉션(List, Set, Array 등)은 기본값을 null로 반환하는 것보다는 빈 컬렉션을 반환할 수 있어요.
컬렉션의 이러한 장점을 살려서, 빈 컬렉션을 반환하도록 변경해보면 어떨까요?